controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentTrigger.java

```diff
@@ -56,20 +64,23 @@
     private final Duration jobTimeout;
 
     private final static Logger log = Logger.getLogger(DeploymentTrigger.class.getName());
+    private static final int triggeringRetries = 5;
 
     private final Controller controller;
     private final Clock clock;
-    private final DeploymentQueue deploymentQueue;
     private final DeploymentOrder order;
+    private final BuildService buildService;
+    private final Executor executor;
```

Comment:
Not used? In any case, this must shut down when the controller is deconstructed.